packages/app-store/_components/AppCard.tsx

```diff
@@ -49,45 +48,36 @@ export default function AppCard({
   const isPlatform = useIsPlatform();
 
   return (
-    <div
-      className={classNames(
-        "border-subtle",
-        app?.isInstalled && "mb-4 rounded-md border",
-        !app.enabled && "grayscale"
-      )}>
-      <div className={classNames(app.isInstalled ? "p-4 text-sm sm:p-4" : "px-5 py-4 text-sm sm:px-5")}>
-        <div className="flex w-full flex-col gap-2 sm:flex-row sm:gap-0">
-          {/* Don't know why but w-[42px] isn't working, started happening when I started using next/dynamic */}
-          <Link
-            href={`/apps/${app.slug}`}
-            className={classNames(app?.isInstalled ? "mr-[11px]" : "mr-3", "h-auto w-10 rounded-sm")}>
+    <Section className={classNames(!app?.isInstalled && "rounded-xl")}>
+      <Section.Header
+        rawHeading={
+          <div>
+            <div className="flex w-full items-center gap-1">
+              <Section.Title>{app?.name}</Section.Title>
+              {!app?.isInstalled && (
+                <span className="bg-emphasis ml-1 rounded px-1 py-0.5 text-xs font-medium leading-3 tracking-[0.01em]">
+                  {app?.categories[0].charAt(0).toUpperCase() + app?.categories[0].slice(1)}
+                </span>
+              )}
+            </div>
+            <Section.Description>{app?.description}</Section.Description>
+          </div>
+        }
+        iconSlot={
+          <Link href={`/apps/${app.slug}`} className="flex h-8 w-8 items-center justify-center">
             <img
               className={classNames(
                 app?.logo.includes("-dark") && "dark:invert",
-                app?.isInstalled ? "min-w-[42px]" : "min-w-[40px]",
-                "w-full"
+                "max-h-full max-w-full object-contain"
               )}
               src={app?.logo}
               alt={app?.name}
             />
           </Link>
-          <div className="flex flex-col pe-3">
-            <div className="text-emphasis">
-              <span className={classNames(app?.isInstalled && "text-base", "font-semibold leading-4")}>
-                {app?.name}
-              </span>
-              {!app?.isInstalled && (
-                <span className="bg-emphasis ml-1 rounded px-1 py-0.5 text-xs font-medium leading-3 tracking-[0.01em]">
-                  {app?.categories[0].charAt(0).toUpperCase() + app?.categories[0].slice(1)}
-                </span>
-              )}
-            </div>
-            <p title={app?.description} className="text-default line-clamp-1 pt-1 text-sm font-normal">
-              {description || app?.description}
-            </p>
-          </div>
-          <div className="ml-auto flex items-center space-x-2">
-            {app.credentialOwner && !isPlatform && (
+        }>
+        <div>
+          <div>
+            {/* {app.credentialOwner && !isPlatform && (
```

Comment:
This wasnt in designs anywhere - not sure where we need to put this or how important it is? @ciaranha